Fix Args for build

It's currently failing to build with the following output:
```
error[E0599]: no method named `about` found for struct `Arg` in the current scope
 --> gubernator/src/cli/apply.rs:7:14
  |
7 |             .about("The manifest file path")
  |              ^^^^^ method not found in `Arg<'_>`

error[E0599]: no method named `about` found for struct `Arg` in the current scope
 --> gubernator/src/cli/config.rs:7:14
  |
7 |             .about("print debug information verbosely"),
  |              ^^^^^ method not found in `Arg<'_>`
  ```
  
  Suggested fix would be to use the `help` function of `Arg` rather than `about`

--- a/gubernator/src/cli/apply.rs
+++ b/gubernator/src/cli/apply.rs
@@ -4,7 +4,7 @@ pub fn subcommand<'a>() -> App<'a> {
     return App::new("apply").about("Apply a Gubernator manifest").arg(
         Arg::new("file")
             .short('f')
-            .about("The manifest file path")
+            .help("The manifest file path")
             .required(true),
     );
 }

--- a/gubernator/src/cli/config.rs
+++ b/gubernator/src/cli/config.rs
@@ -4,6 +4,6 @@ pub fn subcommand<'a>() -> App<'a> {
     return App::new("config").about("Manage Gubernator config").arg(
         Arg::new("debug")
             .short('d')
-            .about("print debug information verbosely"),
+            .help("print debug information verbosely"),
     );
 }

--- a/gubernator/src/generated/greeter.rs
+++ b/gubernator/src/generated/greeter.rs
@@ -43,7 +43,7 @@ pub mod greeter_client {
             interceptor: F,
         ) -> GreeterClient<InterceptedService<T, F>>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
             T: tonic::codegen::Service<
                 http::Request<tonic::body::BoxBody>,
                 Response = http::Response<
@@ -115,7 +115,7 @@ pub mod greeter_server {
         }
         pub fn with_interceptor<F>(inner: T, interceptor: F) -> InterceptedService<Self, F>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
         {
             InterceptedService::new(Self::new(inner), interceptor)
         }

--- a/gubernator_cri/src/generated/cri.rs
+++ b/gubernator_cri/src/generated/cri.rs
@@ -1598,7 +1598,7 @@ pub mod runtime_service_client {
             interceptor: F,
         ) -> RuntimeServiceClient<InterceptedService<T, F>>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
             T: tonic::codegen::Service<
                 http::Request<tonic::body::BoxBody>,
                 Response = http::Response<
@@ -2023,7 +2023,7 @@ pub mod image_service_client {
             interceptor: F,
         ) -> ImageServiceClient<InterceptedService<T, F>>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
             T: tonic::codegen::Service<
                 http::Request<tonic::body::BoxBody>,
                 Response = http::Response<
@@ -2293,7 +2293,7 @@ pub mod runtime_service_server {
         }
         pub fn with_interceptor<F>(inner: T, interceptor: F) -> InterceptedService<Self, F>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
         {
             InterceptedService::new(Self::new(inner), interceptor)
         }
@@ -3136,7 +3136,7 @@ pub mod image_service_server {
         }
         pub fn with_interceptor<F>(inner: T, interceptor: F) -> InterceptedService<Self, F>
         where
-            F: FnMut(tonic::Request<()>) -> Result<tonic::Request<()>, tonic::Status>,
+            F: tonic::service::Interceptor,
         {
             InterceptedService::new(Self::new(inner), interceptor)
         }

